src/main/java/com/example/bank/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,70 @@
+package com.example.bank.exception;
+
+import jakarta.annotation.Nullable;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.util.WebUtils;
+
+import java.util.Collections;
+import java.util.List;
+
+@RestControllerAdvice
+/** 1) Nie widzę różnicy między @RestControllerAdvice i @ControllerAdvice.
+ * Teoretycznie teraz powinienem dostać odpowiedź JSON lub XLM, ale nie zauważyłem różnicy w odpowiedzi z serwera.
+ *
+ * 2) Swoją drogą, skoro zwracana wartość ma być web response, to czy powinienem wrapować poniższe metody
+ *   w ResponseEntity<> ?? W controllerze też ustawiam statusy.
+ *
+ * 3) A po trzecie, czy ktoś te statusy na froncie w ogóle czyta?  </>*/
+public class GlobalExceptionHandler {
+    @ExceptionHandler(
```

Comment:
raczej bym to dal w jedna linijke, zle wyglada to rozwleczone na 3 linie kodu

Reply:
poprawiono